include/bluetooth/uuid.h

```diff
@@ -453,6 +457,54 @@ struct bt_uuid_128 {
  *  @brief Central Address Resolution Characteristic
  */
 #define BT_UUID_CENTRAL_ADDR_RES          BT_UUID_DECLARE_16(0x2aa6)
+/** @def BT_UUID_OTS_FEATURE
+ *  @brief OTS Feature Characteristic
+ */
+#define BT_UUID_OTS_FEATURE               BT_UUID_DECLARE_16(0x2abd)
+/** @def BT_UUID_OTS_OBJECT_NAME
+ *  @brief OTS Object Name Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_NAME           BT_UUID_DECLARE_16(0x2abe)
+/** @def BT_UUID_OTS_OBJECT_TYPE
+ *  @brief OTS Object Type Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_TYPE           BT_UUID_DECLARE_16(0x2abf)
+/** @def BT_UUID_OTS_OBJECT_SIZE
+ *  @brief OTS Object Size Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_SIZE           BT_UUID_DECLARE_16(0x2ac0)
+/** @def BT_UUID_OTS_OBJECT_FIRST_CREATED
+ *  @brief OTS Object First-Created Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_FIRST_CREATED  BT_UUID_DECLARE_16(0x2ac1)
+/** @def BT_UUID_OTS_OBJECT_LAST_MODIFIED
+ *  @brief OTS Object Last-Modified Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_LAST_MODIFIED  BT_UUID_DECLARE_16(0x2ac2)
+/** @def BT_UUID_OTS_OBJECT_ID
+ *  @brief OTS Object ID Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_ID             BT_UUID_DECLARE_16(0x2ac3)
+/** @def BT_UUID_OTS_OBJECT_PROPERTIES
+ *  @brief OTS Object Properties Characteristic
+ */
+#define BT_UUID_OTS_OBJECT_PROPERTIES     BT_UUID_DECLARE_16(0x2ac4)
+/** @def BT_UUID_OTS_OACP
+ *  @brief OTS Object Action Control Point Characteristic
+ */
+#define BT_UUID_OTS_OACP                  BT_UUID_DECLARE_16(0x2ac5)
```

Comment:
I'm wondering if it would be better to only abbreviate control point.
`BT_UUID_OTS_OBJECT_ACTION_CP`
`BT_UUID_OTS_OBJECT_LIST_CP`
`BT_UUID_OTS_OBJECT_LIST_FILTER`

Reply:
yes, sure, it seems like a good idea

Reply:
Thanks. Was `BT_UUID_OTS_OBJECT_LIST_FILTER` too long, since you remove `_OBJECT_` from it?
It would be consistent with the rest of the names to include it.

Reply:
Here is naming in the first version of this PR: https://github.com/zephyrproject-rtos/zephyr/commit/703fc4b8082a8e43f808c403ac68f9f6916c7657

Yes, I missed OBJECT_ part when correcting it, I will fix it soon :)

Reply:
Well O in OTS already stands for Object so we might as well omit it since otherwise it ends up in all the definitions.

Reply:
Sure, I can shorten it

Reply:
@Vudentz, please recheck naming